Frontend/android-native/app/src/driver/yatriPartner/res/values/strings.xml

```diff
@@ -1,8 +1,8 @@
 <resources>
-    <string name="app_name" translatable="false">Yatri Partner</string>
+    <string name="app_name" translatable="false">Yatri Driver</string>
     <string name="client_id" translatable="false">open-kochi</string>
```

Comment:
@Anuragini-2506 can you change the client id to `yatriprovider` for yatri driver and `yatriconsumer` for Yatri customer

Reply:
Also do the same changes for service as well.

Reply:
done